### DISCUSSION TITLE
Generating Typescript types-only package

### DISCUSSION THREAD

**Discussion by xtina** [openapi-ts/openapi-typescript#1432] [Q&A]
Hello, I was wondering if it was possible to generate a Typescript types package out of the box with the generator? I am fairly well versed in the mustache templates, so happy to hack something together as well.

**Comment by drwpow** [comment]
Very doable! GitHub’s Octokit uses openapi-typescript and publishes several types-only packages here: https://github.com/octokit/openapi-types.ts. Just copying any of the `packages/*` should be a good starting point.

**Reply by xtina** [reply]
I posted this question in the wrong repo! 🤦🏻 Thank you for the pointers :)
